### DISCUSSION TITLE
Configuration not shown anymore

### DISCUSSION THREAD

**Discussion by SciLor** [jomjol/AI-on-the-edge-device#2646] [General]
### The Problem

The Configuration page is blank.
Log shows nothing fancy.

### Version

15.3.0

### Logfile

```shell
[32d10h42m39s] 2023-10-03T09:04:45	<ERR>	[POSTPROC] main: Raw: 00294.5937, Value: , Status: Neg. Rate - Read:  - Raw: 00294.5937 - Pre: 294.7937
```

### Expected Behavior

Show config

### Screenshots

![Screenshot_20231003-090418](https://github.com/jomjol/AI-on-the-edge-device/assets/1385855/83979432-3ef6-44a4-9599-0823ee20c558)

### Additional Context

Config worked in the past

**Comment by caco3** [comment]
what happens if you (force-) reload the page?

**Comment by SciLor** [comment]
> what happens if you (force-) reload the page?

I afraid it stay the same.
It worked fine in the beginning and stopped one day.

**Comment by caco3** [comment]
what do the dev tools of your webbrowser say?

you could try to reinstall, just run an OTA update with the latest version.

**Comment by SciLor** [comment]
The devtools show nothing special, no error.
The network stream has no error
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/1385855/fbafb443-6f9a-4d00-8508-e1255e018abc)
If I check the result of the edit_config_param.html it looks fine, except the configuration divall is still on display:none

I would like to investigate the problem itself a bit further, Where to look for the code that shows the divall?

**Comment by SciLor** [comment]
The html look broken, thats why I find no js. So I'll try to copy over the SD files later on.

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/1385855/67d397f6-77b0-4654-a5d8-6f9bfbc09685)

**Comment by caco3** [comment]
Do not copy it over, this will not work.
just run an OTA update with the latest version.

**Reply by SciLor** [reply]
Does the OTA also writes the SD Card content? Iirc in the Installation guide this is a normal process.

**Reply by caco3** [reply]
yes

**Comment by vmojzis** [comment]
Got the same issue with the latest release.

**Reply by SybexX** [reply]
<img width="1028" height="922" alt="Unbenannt" src="https://github.com/user-attachments/assets/c376ba8c-d61a-406d-b459-21d0ebebdd89" />
